webpack/webpack.config.dev-client.js

```diff
@@ -84,11 +84,10 @@ module.exports = {
       publicPath: publicPath
     },
     module: {
-      loaders: commonLoaders.concat([
-        { test: /\.css$/,
-          loader: 'style!css?module&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'
-        }
-      ])
+      loaders: commonLoaders.concat({
+        test: /\.css$/,
+        loader: 'style!css?module&localIdentName=[name]__[local]___[hash:base64:5]!postcss-loader'
```

Comment:
Hahaha! @ZeroCho you're the best! I've never noticed the `s`.

I did however read about the `importLoaders` bit. I wasn't sure what it meant so I left it out, given the below, I think it's safe to add them in. 

> The query parameter importLoaders allow to configure which loaders should be applied to @imported resources.

> importLoaders (int): That many loaders after the css-loader are used to import resources.
 (I initially even tried using `postcss-modules`, but was a bit hard basket at that time. )

The reason why our `import` s were working was because we were using `postcss-import` instead of `css-loader`!